docs/guide/dev/deep_dive/errors/README.md

```diff
@@ -42,6 +42,24 @@ This article contains the list of CKEditor 4 error codes and their explanations.
 * Description: The {@linkapi CKEDITOR.config.cloudServices_uploadUrl} configuration variable for the [CKEditor Cloud Services](https://ckeditor.com/cke4/addon/cloudservices) plugin was not specified.
 * Additional data: None.
 
+## editor-delayed-creation
+
+* Location: `core/editor.js`
+* Description: Editor creation {@linkapi CKEDITOR.config.delayIfDetached has been delayed} because its target native element was detached from DOM.
```

Comment:
```suggestion
* Description: The editor creation {@linkapi CKEDITOR.config.delayIfDetached has been delayed} because its target native element was detached from DOM.
```